localparticipant.go

```diff
@@ -674,3 +678,140 @@ func (p *LocalParticipant) cleanup() {
 	p.rpcPendingAcks = &sync.Map{}
 	p.rpcPendingResponses = &sync.Map{}
 }
+
+func (p *LocalParticipant) StreamText(options StreamTextOptions) *TextStreamWriter {
+	if options.StreamId == nil {
+		streamId := uuid.New().String()
+		options.StreamId = &streamId
+	}
+
+	if options.Attributes == nil {
+		options.Attributes = make(map[string]string)
+	}
+
+	info := TextStreamInfo{
+		baseStreamInfo: &baseStreamInfo{
+			Id:       *options.StreamId,
+			MimeType: "text/plain",
+			Topic:    options.Topic,
+			// Q: Is this correct?
```

Comment:
cc: @lukasIO for confirmation on correctness of this timestamp.

Reply:
yeah timestamp looks good

Reply:
there's also `time.Now().UnixNano()` so wanted to be sure about which one to use.

Reply:
Ideally, we can move to higher resolution timestamps everywhere. If we can use nano everywhere, would be good to do that.

Reply:
letting clients set the timestamp isn't ideal anyways. JS only has ms, so I'd say we leave it at that for now